eclipse-sprotty/sprotty-website
# Jiden/sprotty showcase

example based following on ASCET_Tqs diagram from ETAS
![grafik](https://user-images.githubusercontent.com/34068281/218412861-5c7c89a3-a35d-471a-ab4b-a59f97bf0de2.png)

**How to test**
in main directory:
1. `npm install`
2. `npm run build --workspaces`
3. `npm start`
4. example should be available at `http://localhost:8080/`
